src/actors/FilterBarActor.js

```diff
@@ -0,0 +1,88 @@
+import * as SharedUtils from '../utils/SharedUtils';
+
+export class FilterBarActor {
```

Comment:
The reason the actors are passed through as props is they need to be set up with their dependencies somewhere so they have access to the stores and whatever else without using singletons everywhere